java/src/jmri/jmrit/operations/trains/excel/TrainCustomCommon.java

```diff
@@ -52,6 +52,7 @@ public int getFileCount() {
      * Adds one CSV file path to the collection of files to be processed.
      *
      */
+    @edu.umd.cs.findbugs.annotations.SuppressFBWarnings(value = "UW_UNCOND_WAIT")
```

Comment:
These need justification statements. IIRC it was determined on the developer's mailing list that unjustified suppressed warnings will be periodically removed.